kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/utils/KafkaUtils.java

```diff
@@ -95,7 +95,7 @@ public static String consumeEncryptedMessageWithTestClients(String deployNamespa
         return consumeMessages(deployNamespace, topicName, bootstrap, numOfMessages, "key: kroxylicious.io/encryption", timeout);
     }
 
-    private static String getPodNameByLabel(String deployNamespace, String labelKey, String labelValue, Duration timeout) {
+    public static String getPodNameByLabel(String deployNamespace, String labelKey, String labelValue, Duration timeout) {
         await().atMost(timeout).until(() -> {
```

Comment:
use the two argument form of `until()` to avoid the extra invocation.

Reply:
done